src/wp-includes/cache.php

```diff
@@ -102,6 +102,110 @@ function wp_cache_flush() {
 
 	return $wp_object_cache->flush();
 }
+/**
+ * Removes all cache items in a group.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ *
+ * @return bool True on success, false on failure group not found.
+ */
+function wp_cache_flush_group( $group, $clear_linked = true ) {
+
+	global $wp_object_cache;
+
+	if ( ! wp_cache_supports_flushing_groups() ) {
+
+		return false;
+	}
+
+	// if group is an array loop and call each key in the array
+	if ( is_array( $group ) ) {
+
+		array_map( 'wp_cache_flush_group', array_values( $group ) );
+
+		return true;
+	}
+
+	// these are linked cache groups, so we have to flush them both if one is called
+	$cache_pairs = wp_cache_get_linked_meta();
+	if ( $clear_linked && isset( $cache_pairs[ $group ] ) ) {
+		foreach ( $cache_pairs[ $group ] as $cache_pair ) {
+			wp_cache_flush_group( $cache_pair, false );
+		}
+	}
+
+	/**
+	 * Filters to allow caching plugins to set this function to be called to flush groups.
+	 * See: filter "wp_cache_supports_flushing_groups" to declare group flushing is supported.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param string return $flush_group_function name of function to call from the cashing class.
+	 */
+	$flush_group_function = apply_filters( 'wp_cache_flush_group_function', 'flush_group' );
```

Comment:
Not a function, but a method right?

Reply:
What is the benefit of this filter?

Reply:
this allowed plugins to change function that handles the flushing 
removed